wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+Users are getting banned for several reasons not limited to:
+-   Continuous misbehaviour regardless of how many infringements were given.
+-   Sharing your account with other people.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
+The rules says that any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule; cheaters will be *permanently banned*.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
+## I have been banned for cheating. I didn't cheat!
 osu! uses an automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The staff never places an incorrect ban for cheating. Before they take any actions, they make sure it first 100%.
 
-My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
---------------------------------------------------------------------------------------
 
+## My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
 It may be hard to come to terms with, but you will have to trust the staff. peppy is not trying to tear apart friendships but he knows that nearly no one can be trusted. Even those people who are close to you that you may never suspect can be just acting behind you or any other person.
```

Comment:
replace "staff" with "team"

I'm not sure this whole thing is good, honestly. I'll try to come up with a better way to say all of that.

Reply:
I'll wait for it.

Reply:
Try this:

"We understand that you trust your friends, but you will have to believe the team. Punishments aren't issued unless there is sufficient evidence to support the decision. Whether or not they have cheated, it shouldn't affect your friendship!"